server/src/test/java/org/apache/druid/server/coordinator/rules/BroadcastDistributionRuleTest.java

```diff
@@ -315,19 +315,19 @@ public void testBroadcastToSingleDataSource()
 
   /**
    * Servers:
-   * name         | segments
-   * -------------+--------------
-   * general      | large segment
-   * maintenance1 | small segment
-   * maintenance2 | large segment
+   * name             | segments
+   * -----------------+--------------
+   * general          | large segment
```

Comment:
Probably should be "active" to match the terminology in production code.

Reply:
good point, fixed